src/types/claim/resolvers.js

```diff
@@ -19,6 +19,12 @@ module.exports = {
   Claim: {
     references: (_) => {
       return _.references;
+    },
+    qualifier: (_, { propertyId }) => {
+      if (_.qualifiers ) {
+        return _.qualifiers[propertyId];
+      }
+      return null;
```

Comment:
It seems cleaner to me to always return a list, i.e. return `[]` here, not  `null`. I'm not sure if there's a way to express in the schema that the list is non-nullable, but allowed to be empty.